src/wp-includes/http.php

```diff
@@ -581,7 +581,21 @@ function wp_http_validate_url( $url ) {
 	}
 
 	$port = $parsed_url['port'];
-	if ( 80 === $port || 443 === $port || 8080 === $port ) {
+
+
+	/**
+	 * Controls list of ports considered safe in HTTP Api
```

Comment:
Slight grammatical changes and API changed to uppercase for consistency:
```suggestion
	 * Controls the list of ports considered safe in the HTTP API.
```